src/quo2/components/navigation/page_nav/view.cljs

```diff
@@ -20,7 +20,7 @@
    :blur        :grey})
 
 (defn- page-nav-base
-  [{:keys [margin-top background on-press accessibility-label icon-name]
+  [{:keys [margin-top background on-press accessibility-label icon-name overlay-shown?]
```

Comment:
seems like a good solution, perhaps it needs a commit to explain why it is needed? 🤔

Reply:
You mean in the commit message or a comment?

Reply:
Just in case it's a comment, I'll do it now

Reply:
also a nit: perhaps `behind-overlay?` is a clearer name for this prop?  🤨

Reply:
Renamed

Reply:
Yep I meant in a comment